README.md

```diff
@@ -27,6 +27,19 @@ Then create a file named `.eslintrc` with following contents in the root folder
 
 You can override the settings from `eslint-config-sku` by editing the `.eslintrc` file. Learn more about [configuring ESLint](http://eslint.org/docs/user-guide/configuring) on the ESLint website.
 
+## sku/unpretty
+
+The core sku config does not include formatting concerns.  
+It is recommended to use Prettier (https://www.npmjs.com/package/prettier) to lint formatting concerns.
+If you are not using prettier and would like to enable formatting use `sku/unpretty` in addition to the core `sku` config.
```

Comment:
Nit picks: Prettier should be capitalised, and "formatting" should be followed by a comma.

Reply:
Also, should it be "enable formatting rules" rather than just "enable formatting"?